src/docs/asciidoc/security.adoc

```diff
@@ -218,94 +219,124 @@ Hazelcast provides a default SSLContextFactory, `com.hazelcast.nio.ssl.BasicSSLC
 ```
- 
+
+This way of TLS/SSL configuration is then system wide in your Java runtime.
 
 ==== TLS/SSL for Hazelcast Clients
 
-Hazelcast clients which support TLS/SSL should have the following properties to configure the TLS/SSL:
+The TLS configuration in Hazelcast clients is very similar to member configuration.
 
-```
-Properties clientSslProps = TestKeyStoreUtil.createSslProperties();
-clientSslProps.setProperty("javax.net.ssl.trustStore", getTrustStoreFilePath());
-clientSslProps.setProperty("javax.net.ssl.trustStorePassword", "123456");
-ClientConfig config = new ClientConfig();
-config.getNetworkConfig().setSSLConfig(new SSLConfig().setEnabled(true).setProperties(clientSslProps));
+```xml
+<hazelcast-client>
+  ...
+  <network>
+    ...
+    <ssl enabled="true">
+      <factory-class-name>
+          com.hazelcast.nio.ssl.BasicSSLContextFactory
+      </factory-class-name>
+      <properties>
+        <property name="keyStore">/opt/hazelcast-client.keystore</property>
+        <property name="keyStorePassword">clientsSecret</property>
+        <property name="trustStore">/opt/hazelcast-client.truststore</property>
+        <property name="trustStorePassword">changeit</property>
+        <property name="protocol">TLSv1.2</property>
+      </properties>
+    </ssl>
+  </network>
+  ...
+</hazelcast-client>
 ```
 
+The same `BasicSSLContextFactory` properties used for members are available on clients.
+Clients don't need to set `mutualAuthentication` property as it's property configuring server side of TLS connections.
+
 ==== Mutual Authentication
 
-As explained above, Hazelcast members have keyStore used to identify themselves (to other members) and Hazelcast clients have trustStore used to define which members they can trust. 
+TLS connections have two sides - one which opens the connection (i.e. TLS client)
+and other one which accepts the connection (TLS server).
+By default only the TLS server proves its identity by presenting a certificate to the TLS client.
+The Mutual authentication means that also TLS clients prove their identity to TLS servers.
 
-Starting with Hazelcast 3.8.1, mutual authentication is introduced. This allows the clients also to have their keyStores and members to have their trustStores so that the members can know which clients they can trust.
+Hazelcast members can be on both sides of TLS connection - TLS servers and TLS clients.
+Hazelcast clients are always on client side of a TLS connection.
 
-To enable mutual authentication, you need to set the following property at the member side:
+By default Hazelcast members have keyStore used to identify themselves to clients and other members.
+Both Hazelcast members and Hazelcast clients have trustStore used to define which members they can trust.
 
-```
-props.setProperty("javax.net.ssl.mutualAuthentication", "REQUIRED");
-```
+When the Mutual authentication feature is enabled, Hazelcast clients need to have keyStore provided.
```

Comment:
When the mutual authentication feature is enabled, Hazelcast clients need to provide keyStore.